### PR TITLE
Use the sandbox domain for the tutor iframe

### LINKED ISSUE #2601: Move the tutor iframe to a sandboxed domain
We don't have a lot of faith in the JS code of the python tutor. Since it's shown in an iframe already, it would be good to host it from a separate domain, similar to exercise descriptions.

### PR DESCRIPTION
**Pull request by bmesuere**
This pull request adds the sandbox host to the global `dodona` JS object and uses that value to create the tutor iframe. No changes were made to the routes because the tutor is a bunch of html pages in the public map.

This should reduce the impact of xss bugs in the tutor code.

I tested this on the 3 pages where we use the tutor:
- the bottom of an exercise page
- a submission page
- an evaluation page

Downside: the tutorviz file uses a manual copy of the iframe lib. This will not be updated automatically by dependabot.

Closes #2601

### CODE REVIEW

**Comment by bmesuere** [comment]
2 issues remaining:
- 1 not related with this PR: the sandbox url on mestra is set to mestra.ugent.be and we have no idea why
- loading the tutor from the sandbox domain fails because we call the load of the iframe from the parent page and this is not allowed. The question remains if we can manually allow this.

**Comment by chvp** [comment]
The sandbox url is configured in the apache VHost using passenger_env. It was set to mestra.ugent.be because we did not yet have a sandbox domain at that time.

**Comment by bmesuere** [comment]
This is ready for review and deployed on naos for testing. For now, I enabled the logging to console, but this should be removed before merging. I manually tested using chrome on the 3 pages where we display submissions.

There are a number of issues/strange things I encountered when developing this:
- The mestra sandbox domain (mestra-sandbox.dodona.be) was not added to the https certificate (@rien)
- To get this working, the iframe resize lib needs to be loaded on the parent page. After trying several things, the code to get this working was adding `require("iframe-resizer");` to the pythia submission pack, similar to how this was added to the exercise pack. However, the existing code contains 2 other ways of adding this library which didn't work for me. They may not work in the current code either be be leftovers of previous experiments:
  - The description pack contains `import { iframeResizerContentWindow } from "iframe-resizer";` and `window.iframeResizerContentWindow = iframeResizerContentWindow;` however we never call this function anywhere.
  - The series overview pack contains `import { iframeResizer } from "iframe-resizer";`. Like I said, this syntax didn't work for me.

**Comment by chvp** [comment]
The descriptions on the series overview page are indeed broken, so that import should probably be changed as well. (Shows how much that page is used, if we didn't know this was broken for a year.)

The `iframeResizerContentWindow` is required by iframe-resizer to be able to resize the iframe. See https://github.com/davidjbradshaw/iframe-resizer/blob/master/docs/troubleshooting.md#iframe-not-resizing. Maybe a comment could be added? I'm not sure if the `window` assignation is required or just the import.

**Review by chvp — APPROVED** [review]

**Comment by pdawyndt** [comment]
@chvp One of the use-cases for the series overview page was printing all exercises in a series at once, but that feature itself has been broken since the introduction of the description iframes (#1421). Chain reaction.

**Review by niknetniko — APPROVED** [review]

File: public/tutorviz/javascript/iframe.min.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*! iFrame Resizer (iframeSizer.contentWindow.min.js) - v4.3.2 - 2021-04-26
```
Comment (chvp) [inline public/tutorviz/javascript/iframe.min.js:1]: Actually, if this file has to be vendored (and I don't see a better solution immediately), maybe a comment should be added to package.json to update this file if the iframe-resizer dependency is updated.

Comment (bmesuere) [reply public/tutorviz/javascript/iframe.min.js:1]: JSON doesn't support comments :(

Comment (chvp) [reply public/tutorviz/javascript/iframe.min.js:1]: Right, that's unfortunate.

**Pull request merged**